server/src/lib.rs

```diff
@@ -148,6 +152,12 @@ impl Run {
     }
 }
 
+async fn index_graphiql() -> Result<HttpResponse> {
```

Comment:
question: This provides both the playground UI for graphql *and* the actual endpoint?

Reply:
Yes it does, I think it uses a combo of GET, empty query and client being a browser to determine it's not a query/mutation to serve the playground.